src/inetstack/protocols/layer4/tcp/peer.rs

```diff
@@ -109,7 +109,7 @@ impl SharedTcpPeer {
         // All other checks should have been done already.
         debug_assert!(!Ipv4Addr::is_unspecified(local.ip()));
         debug_assert!(local.port() != 0);
-        debug_assert!(self.addresses.get(&SocketId::Passive(local)).is_none());
```

Comment:
We have a few of these in the runtime/mod.rs file as well.

Reply:
There is 1 instance in the runtime/mod.rs, but that is because the IDMap does not have a `contains_key` method. We can change this in a separate PR if required.